### PR TITLE
Re-factor(prl-3811 : fixed issues on page titles for confirm contact …

### PR DESCRIPTION
**Pull request by SasankAchyuta9**
…details , proceedings , confidentiality error for respondent , documents issue for opening in parent window)

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/respondent/yourdocuments/alldocuments/medicalreports/template.njk [hunk 1]
```diff
@@ -21,7 +21,7 @@
               {{doc.createdDate}}
             </dt>
             <dd class="govuk-summary-list__value">
-              <a href={{doc.href}}>{{ doc.fileName }}</a>
```
Comment (vivek-sekhar) [inline src/main/steps/respondent/yourdocuments/alldocuments/medicalreports/template.njk:24]: why are there lot of templates repeating  the same code and duplicated? use one template instead

File: src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.test.ts [hunk 2]
```diff
@@ -457,7 +457,7 @@ describe('ProceedingDetailsMapper', () => {
   });
 
   test('Should return value for providing a date for getDisplayDate', () => {
-    const retunrn_date = { day: undefined, month: undefined, year: 'May' };
+    const retunrn_date = { day: undefined, month: undefined, year: 'June' };
```
Comment (vivek-sekhar) [inline src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.test.ts:460]: Does this work at all?

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/respondent/yourdocuments/alldocuments/medicalreports/template.njk (hunk 1 above)
Comment (SasankAchyuta9) [reply src/main/steps/respondent/yourdocuments/alldocuments/medicalreports/template.njk:24]: Yes vivek created one common template for all docs of respondent , applicant. Placed it in common folder as ... common/alldocuments/template.njk

File: src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.test.ts (hunk 2 above)
Comment (vivek-sekhar) [reply src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.test.ts]: Can u change the system date to future and test pls and confirm?

File: src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.test.ts [hunk 3]
```diff
@@ -457,8 +457,8 @@ describe('ProceedingDetailsMapper', () => {
   });
 
   test('Should return value for providing a date for getDisplayDate', () => {
-    const retunrn_date = { day: undefined, month: undefined, year: 'June' };
-    expect(getDisplayDate(new Date())).toEqual(retunrn_date);
+    const retunrn_date = { day: '20', month: '03', year: '2023' };
```
Comment (vivek-sekhar) [inline src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.test.ts:460]: Remove this as it must gave been fixed already in 874PR

File: src/main/steps/common/alldocuments/template.njk [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{% extends "common/template.njk" %}
```
Comment (vivek-sekhar) [inline src/main/steps/common/alldocuments/template.njk:1]: where this new template is used?

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/common/alldocuments/template.njk (hunk 4 above)
Comment (SasankAchyuta9) [reply src/main/steps/common/alldocuments/template.njk:1]: Its been used in all the documents templates. for both applicant and respondent.

File: src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.test.ts (hunk 3 above)
Comment (SasankAchyuta9) [reply src/main/steps/tasklistresponse/proceedings/ProceedingDetailsMapper.test.ts]: Yes Vivek , this  is removed.

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**